src/child/index.js

```diff
@@ -30,6 +30,12 @@ currentWindowService.ready(() => {
     const windowStateService = new WindowStateService(currentWindow, store);
     windowStateService.start();
 
+    currentWindow.addEventListener('bounds-changed', (e) => {
+        if (e.changeType && !store.getState().childWindows[e.name].windowState.isCompact) {
```

Comment:
I think `e.changeType !== 0` would show the intent more clearly, maybe with a comment explaining that this is a change in position, not size. Or even `const positionChangeType = 0;` and `e.changeType !== positionChangeType`